cfgov/v1/tests/wagtail/test_organisms.py

```diff
@@ -0,0 +1,224 @@
+from django.test import TestCase
+from django.test import Client
+
+
+from scripts import _atomic_helpers as atomic
+from v1.models.landing_page import LandingPage
+from v1.models.sublanding_page import SublandingPage
+from v1.models.learn_page import LearnPage
+from v1.models.browse_page import BrowsePage
+from wagtail.wagtailcore.blocks import StreamValue
+from helpers import publish_page
+from v1.models.snippets import Contact
+
+c = Client()
+
+'''
+TODO: Create tests for the following organisms:
+          - FilterableListControls
+          - Sidebar Breakout
+          - PostPreviewSnapshot
+          - RelatedPosts
+'''
+
+
+class OrganismsTestCase(TestCase):
+	def get_contact(self):
+		contact = Contact(heading='Test User')
+		contact.contact_info = StreamValue(
+			contact.contact_info.stream_block,
+			[
+				atomic.contact_email,
+				atomic.contact_phone,
+				atomic.contact_address
+			], 
+			True
+		)
+		contact.save()
+		return contact 
+
+	def test_well(self):
+		"""Well content correctly displays on a Landing Page"""
+		landing_page = LandingPage(
```

Comment:
@vickumar1981 I understand the urge to abstract this and I think it makes sense in most cases to do what you're saying. In this case, though, these tests may appear to be duplicative but they're not.  The name of the slug could be anything; it's only dependent on the test data I create, which is created in each individual test.  I think I could have a few more helper methods, like "initialize_landing_page" for pages that are created more than once.